docker/docker-compose.yaml

```diff
@@ -167,6 +167,8 @@ services:
     build:
       context: ..
       dockerfile: sapling-dev-server/Dockerfile
+      args:
```

Comment:
typo in commit message: enviromental -> environment (in two places)

Reply:
Fixed!